valohai_yaml/schema/input-item.yaml

```
@@ -38,3 +38,6 @@ properties:
           - "suffix"
     default: false
     description: Whether to retain directories when using wildcards for this input.
+  overridable:
+    type: boolean
+    description: Whether to replace or append the input when something else is overriding it.
```

I think the description is at odds with the name of the property here, and since we'll be stuck forever with what we decide here, it'd better be good...

```suggestion
  override-mode:
    type: string
    enum:
      - replace
      - append
    default: append
    description: Whether to replace the values set for this input, or to append to them when overridden by e.g. a pipeline edge.
```
(Is the description correct?)

---

yes, and `default: append ` wouldn't be effective when pipeline node explicitly define `override`. Linter will throw error that append-able input and explicitly overriding inputs with edge are mutually exclusive.
As explicit override will definitely replace the input.